tools/python/dlib/__init__.py.in

```diff
@@ -1,20 +1,5 @@
 # Copyright (C) 2020  Davis E. King (davis@dlib.net)
 # License: Boost Software License   See LICENSE.txt for the full license.
 
-def add_lib_to_dll_path(path):
-    """ On windows you must call os.add_dll_directory() to allow linking to external DLLs.  See
-    https://docs.python.org/3.8/whatsnew/3.8.html#bpo-36085-whatsnew.  This function adds the folder
-    containing path to the dll search path. 
-    """
-    try:
-        import os
-        os.add_dll_directory(os.path.join(os.path.dirname(path), '../../bin'))
-    except (AttributeError,KeyError,FileNotFoundError):
-        pass
-
-if '@DLIB_USE_CUDA@' == 'ON':
-    add_lib_to_dll_path('@cudnn@')
-    add_lib_to_dll_path('@CUDA_CUDART_LIBRARY@')
```

Comment:
I have to imagine we still need this for windows users.  Did you test this stuff on windows?

Reply:
We don’t need it for conda-forge, I’m not sure for vanilla builds